### PR TITLE
Add glinet integration

### PR DESCRIPTION
**Pull request by HarvsG**
## Proposed change

Addition for a new integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/53062/
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2696
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/glinet.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+You need to know your router's IP address and login password - the defualts for GL-inet routers are `192.168.8.1` and `goodlife`. For security reasons you should change your password to something more secure.
+
+</div>
+
+To add devices to track into your Home Assistant installation, go to:
+
+**Configuration** -> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **GL-inet**. Then complete the pop up with your router address in the format `http://192.168.0.1` and the password.
+**Note**: The router password **is** stored by HomeAssistant as the API access tokens will expire
+
```
Comment (frenck) [inline source/_integrations/glinet.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/glinet.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+GL-inet are a Hong-Kong based company who produce highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
+
+There is currently support for the following device types within Home Assistant:
+
+- **Presence Detection** - The GL-inet platform offers presence detection by looking at connected devices to a GL-inet based router.
+
+Upder development:
+- **Sensor** - The GL-inet sensor platform could allow you to get upload and download data from your router within Home Assistant.
+- **Switch** - The GL-inet switch platform could allow you to activate and deactivate your VPN from homeassistant
+
```
Comment (frenck) [inline source/_integrations/glinet.markdown]: We don't list these in our documentation.

```suggestion
```

File: source/_integrations/glinet.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@HarvsG'
+ha_domain: glinet
+ha_platforms:
+  - device_tracker
+---
+
+GL-inet are a Hong-Kong based company who produce highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
```
Comment (frenck) [inline source/_integrations/glinet.markdown]: Can we link them in the text?

```suggestion
GL-inet is a Hong-Kong based company that produces highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
```

File: source/_integrations/glinet.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+The intergration will attempt to give each device a readable name like `device_tracker.myIphone`. However if it cannot the device will recieve a name like `device_tracker.b8_27_eb_53_a2_4f`, this is based on the device's unique [MAC address](https://kb.netgear.com/1005/How-do-I-find-my-device-s-MAC-address).
+
+</div>
+
+
+## Integration Options
+
+It is possible to change some behaviors through the integration options. These can be changed at **GL-inet** -> **Configure** on the Integrations page.
+
+- **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home"
```
Comment (frenck) [inline source/_integrations/glinet.markdown]:
```suggestion
{% include integrations/option_flow.md %}
{% configuration_basic %}
Consider home: Number of seconds that must elapse before considering a disconnected device "not at home".
{% endconfiguration_basic %}
```

File: source/_integrations/glinet.markdown [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+**Note**: The intergration will automatically add every now device that connects to your network. Once you have selected the devices you want to track, disable the integration system option `Enable new added entities`
```
Comment (frenck) [inline source/_integrations/glinet.markdown]: Please format notes as per documentation standards:

https://developers.home-assistant.io/docs/documenting/create-page#html

File: source/_integrations/glinet.markdown [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+ha_domain: glinet
+ha_platforms:
+  - device_tracker
+---
+
+GL-inet are a Hong-Kong based company who produce highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
+
+There is currently support for the following device types within Home Assistant:
+
+- **Presence Detection** - The GL-inet platform offers presence detection by looking at connected devices to a GL-inet based router.
```
Comment (frenck) [inline source/_integrations/glinet.markdown]:
```suggestion
There is currently support for the following device types within Home Assistant:

- **Presence Detection** - Presence detection by looking at connected devices to a GL-inet based router.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/glinet.markdown (hunk 3 above)
Comment (HarvsG) [reply source/_integrations/glinet.markdown]: Done in https://github.com/home-assistant/home-assistant.io/pull/18537/commits/ac3fd913b97a91904c1c1791810b24e90c6dac43 and https://github.com/home-assistant/home-assistant.io/pull/18537/commits/051ee9f244f78c3fa9538fb3aa15f585ca0c352c

File: source/_integrations/glinet.markdown (hunk 5 above)
Comment (HarvsG) [reply source/_integrations/glinet.markdown]: https://github.com/home-assistant/home-assistant.io/pull/18537/commits/051ee9f244f78c3fa9538fb3aa15f585ca0c352c

**Review by frenck — CHANGES_REQUESTED** [review]
Please make sure the builds pass before requesting a review.

**Comment by HarvsG** [comment]
Not sure why the GH build is failing, this error is very opaque

```
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Liquid Exception: undefined method `key?' for #<String:0x000055914b4166a8> in /home/runner/work/home-assistant.io/home-assistant.io/source/_integrations/glinet.markdown
                    ------------------------------------------------
      Jekyll 4.2.0   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
Generating site failed
Error: Process completed with exit code 1.
```

Sorry to tag you @frenck, I've got no idea why the build is failing. Could it be something in the template tags?

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**